react/index.js

```diff
@@ -223,6 +223,12 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
+    const getBuyButtonVisibility = () => {
+      return !((showButtonOnHover || showBuyButton) && showBuyButton) ? 'hidden' : 'visible'
```

Comment:
`showButtonOnHover` does not matter on this clause. The visibility only depends on `showBuyButton` in this case. If `true`, 'visible, if not `hidden`.

What's the expected behavior?